azurelinuxagent/ga/collect_logs.py

```diff
@@ -143,13 +162,15 @@ def _get_resource_limits():
         memory_limit = "30M"  # K for kb, M for mb
         return cpu_limit, memory_limit
 
-    @staticmethod
-    def _collect_logs():
+    def _collect_logs(self):
         logger.info("Starting log collection...")
 
         # Invoke the command line tool in the agent to collect logs, with resource limits on CPU and memory (RAM).
-        scope_name = "collect-logs-{0}.scope".format(ustr(int(time.time() * 1000000)))
-        systemd_cmd = ["systemd-run", "--unit={0}".format(scope_name), "--scope"]
+
+        systemd_cmd = [
+            "systemd-run", "--unit={0}".format(logcollector.CGROUPS_UNIT),
+            "--slice={0}".format(cgroupconfigurator.AZURE_SLICE)
+        ]
```

Comment:
Is logcollector demon managed under azure.slice group. May be comment like [this ](https://github.com/Azure/WALinuxAgent/blob/b268011333a50e07db385824ffb963f975513890/azurelinuxagent/common/cgroupconfigurator.py#L254)helps